cordova-lib/spec-cordova/metadata/windows_parser.spec.js

```diff
@@ -115,31 +110,9 @@ describe('windows8 project parser', function() {
             read = spyOn(fs, 'readFileSync').andReturn('');
         });
 
-        describe('update_from_config method', function() {
```

Comment:
Intead of removing these tests could you please update them to test that corresponding script is being required, and the error is thrown when it is missing.